macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -108,7 +108,13 @@ open class AvatarView: NSView {
 			contactImageView.image = contactImage
 
 			// Update our display style
-			displayStyle = contactImage == nil ? .initials : .image
+			if contactImage == nil {
+				displayStyle = .initials
```

Comment:
Does this need accessibility update same as line 342?

Reply:
make sense. I just ended up calling updateAvatarViewContents() instead of doing it directly here.